readme.md

```diff
@@ -2704,6 +2704,18 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   1 - 4 |       10 | 30-45m |
 
+
+### [Path of Civilization](https://boardgamegeek.com/boardgame/385680/path-civilization)
+
+> In Path of Civilization, from the very first second of the game you must make choices that will have repercussions until the end of the game. The game is played simultaneously and works with a simple card and resource management system. The diversity of its cards guarantees enormous variability.
```

Comment:
The linter action thinks the image is part of the quote block. Instead there should be a line between this paragraph's line and the next line.

```suggestion
> In Path of Civilization, from the very first second of the game you must make choices that will have repercussions until the end of the game. The game is played simultaneously and works with a simple card and resource management system. The diversity of its cards guarantees enormous variability.

```